packages/cli/src/lib/webdriver.ts

```diff
@@ -21,6 +21,10 @@ const startDriver = async (
       }, options);
     }
 
+    if (config.chromePath) {
+      options.setChromeBinaryPath(config.chromePath);
```

Comment:
I don't think we should force a user to type an absolute path here we should do something like 
```js
options.setChromeBinaryPath(path.resolve(config.chromePath));
```

Reply:
I made the change and rebased the PR, sorry for the delay